@xen-orchestra/backups/_runners/_vmRunners/IncrementalRemote.mjs

```diff
@@ -50,7 +50,17 @@ class IncrementalRemoteVmBackupRunner extends AbstractRemote {
             }),
           'writer.transfer()'
         )
-        // @todo: should reformat parent name with alias.vhd if vhdfile to vhddirectory, without for vhddir=>vhdfile
+        // this will update parent name with the needed alias
+        await this._callWriters(
+          writer =>
+            writer.updateUuidAndChain({
+              isVhdDifferencing,
+              timestamp: metadata.timestamp,
+              vdis: incrementalExport.vdis,
+            }),
+          'writer.updateUuidAndChain()'
+        )
+
```

Comment:
This runner must call `checkBaseVdis()` on writers, and if anything is missing throw an error.